src/components/UI/SearchBar/index.tsx

```diff
@@ -23,6 +23,17 @@ export default function SearchBar() {
         placeholder={t('search')}
         id="search"
       />
+      <span className="clearSearchInputWrapper">
```

Comment:
is this extra wrapper needed? I think you should be able to position the clickable element directly

Reply:
I believe so, having the padding directly on the element would cause all the space to the left of the element to be clickable, unless you can think of a better way of doing it

Reply:
instead of doing right: 0px with padding, you can do right > 0 and no padding, or you can try with margin instead of padding so it's not part of the click area

Reply:
This should also be resolved now, thanks for the help